src/js/utils/messageHandlers.js

```diff
@@ -74,7 +74,7 @@ export function togglePopup(tab) {
     }
 
     if (isPopupOpen) {
-      sendMessage("closePopup", `content-script@${tab.id}`)
+      sendMessage("closePopup", null, `content-script@${tab.id}`)
```

Comment:
`sendMessage` requires a second parameter for the data. This was missing, otherwise Asana would also have worked after clicking two times on the bubble. But due to this bug, the message to close the popup could not be sent.